guides/v2.1/coding-standards/technical-guidelines/technical-guidelines.md

```diff
@@ -471,6 +471,39 @@ We are reviewing this section and will publish it soon.
 
 13.4. Exception in a single CLI command SHOULD NOT break the CLI framework; running other commands SHOULD still be possible.
 
+## 14. Events
+
+14.1. All values (including objects) passed to event MUST NOT be modified in the event observer. Use plugins instead for this application.
```

Comment:
>passed to event

Please add article: _passed to **an** event_

>Use plugins instead for this application

May I suggest another wording: _Instead, use plugins for this application.